src/getting-started.md

```diff
@@ -96,174 +96,28 @@ See [this chapter][config] for more info about configuration.
 
 [config]: ./building/how-to-build-and-run.md#create-a-configtoml
 
-### Building and Testing `rustc`
+### Common `x.py` commands
 
-Here is a summary of the different commands for reference, but you probably
-should still read the rest of the section:
+Here are the basic invocations of the `x.py` commands most commonly used when
+working on `rustc`, `std`, `rustdoc`, and other tools.
 
 | Command | When to use it |
 | --- | --- |
-| `./x.py check` | Quick check to see if things compile; [rust-analyzer can run this automatically for you][rust-analyzer] |
-| `./x.py build --stage 0 [library/std]` | Build only the standard library, without building the compiler |
-| `./x.py build library/std` | Build just the 1st stage of the compiler, along with the standard library; this is faster than building stage 2 and usually good enough |
-| `./x.py build --keep-stage 1 library/std` | Build the 1st stage of the compiler and skips rebuilding the standard library; this is useful after you've done an ordinary stage1 build to skip compilation time, but it can cause weird problems. (Just do a regular build to resolve.) |
-| `./x.py test [--keep-stage 1]` | Run the test suite using the stage1 compiler |
-| `./x.py test --bless [--keep-stage 1]` | Run the test suite using the stage1 compiler _and_ update expected test output. |
-| `./x.py build --stage 2 compiler/rustc` | Do a full 2-stage build. You almost never want to do this. |
-| `./x.py test --stage 2` | Do a full 2-stage build and run all tests. You almost never want to do this. |
-
-To do a full 2-stage build of the whole compiler, you should run this (after
-updating `config.toml` as mentioned above):
+| `./x.py check` | Quick check to see if most things compile; [rust-analyzer can run this automatically for you][rust-analyzer] |
+| `./x.py build` | Builds `rustc`, `std`, and `rustdoc` |
+| `./x.py test` | Runs all tests |
```

Comment:
I just tried running `x.py setup` followed by `x.py test` on a fresh clone of `rust-lang/rust`. The debuginfo and run-make tests didn't cause any problems. I did hit https://github.com/rust-lang/rust/issues/80096; I then set `download-ci-llvm = false` and re-ran `x.py test` and it worked fine.

So I think suggesting `x.py test` is fine, though it would be good to get https://github.com/rust-lang/rust/issues/80096 fixed. IMO, running all the tests is the best starting point.

Reply:
What profile did you choose for `x.py setup`?

Reply:
(b) compiler

Reply:
Hmm, ok. I'm surprised it worked out of the box, but maybe some stuff was fixed without me noticing :)